### PR TITLE
feat: add-sentry

### LINKED ISSUE #58: Set up error tracking
On launch, we should have an error tracking tool in place. I'm familiar with [Sentry](https://sentry.io/welcome/) and would recommend it.

@agraebe Are you able to set up an account/billing?

### PR DESCRIPTION
**Pull request by agraebe**
Adding Sentry tracking to fix #58

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
+require('dotenv').config();
+
```
Comment (aulneau) [inline next.config.js:2]: With next.js we don't need dotenv configs, see here https://nextjs.org/blog/next-9-4#new-environment-variables-support

File: package.json [hunk 2]
```diff
@@ -17,16 +17,21 @@
...
     "@types/nprogress": "^0.2.0",
     "@types/prismjs": "^1.16.0",
     "@types/styled-components": "^5.1.0",
+    "@zeit/next-source-maps": "0.0.4-canary.1",
     "babel-plugin-module-resolver": "^4.0.0",
     "blockstack": "^21.0.0",
     "c32check": "^1.0.1",
     "cacheable-response": "^2.0.8",
     "dayjs": "^1.8.26",
+    "dotenv": "^8.2.0",
```
Comment (aulneau) [inline package.json:34]: See prev comment, no need for dotenv

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -26,10 +35,15 @@ class MyApp extends App<MyAppProps & ReduxWrapperAppProps<RootState>> {
 
   render() {
     const { Component, pageProps, colorMode, store } = this.props;
+
+    // Workaround for https://github.com/zeit/next.js/issues/8592
+    const { err } = this.props;
```
Comment (aulneau) [inline src/pages/_app.tsx:40]: `err` should be taken our from above, no need to have two instances of destructuring this.props.

File: next.config.js (hunk 1 above)
Comment (agraebe) [reply next.config.js:2]: oh, nice. Nextjs never ceases to amaze me!

**Comment by agraebe** [comment]
@aulneau should we do this before the release? the errors would be a good source for troubleshooting, especially if others might need to jump in

**Comment by aulneau** [comment]
@agraebe it would be nice, do you still want to take it up?

**Comment by agraebe** [comment]
yes, let me resolve the conflicts. did you set the config vars for the Sentry team accounts?

**Comment by aulneau** [comment]
Closing this due to age. We can continue work in a new branch as this is still a pending todo.

**Pull request closed without merging**